helm/examples/azure/log-analytics/README.md

```diff
@@ -6,37 +6,53 @@ a Log Analytics workspace, and explicitly enabling the Log Analytics feature. A
+ New workspace will be created if it does not exist
+- LOGANALYTICS_RESOURCE_GROUP (The Azure resource group associated with the target AKS cluster)
+ New workspace will be associated with this resource group
+- AKS_CLUSTER_NAME (Name of the target AKS cluster to associate log analytics workspace)
+- TAGS - The tags associated with the new workspace
+     For example: "admin=MyName email=my.email@mycompany.com environment=myenv justification=testing"
+- AZURE_SUBSCRIPTION (Optional - Ensures this subscription is set before creating the new workspace)
+
+#### b - Execute enable-loganalytics.sh
+
+This helper script attempts to create new Azure LogAnalytics workspace (user can provide pre-existing), associates the workspace with the target AKS cluster, and enables the Azure Log Analytics feature.This script is dependant on the values provided in the previous step
+
+### 2 - Configure HPCC logAccess
+The logAccess feature allows HPCC to query and package relevant logs for various features such as ZAP report, WorkUnit helper logs, ECLWatch log viewer, etc.
+
+#### a - Procure AAD registered application
+Azure requires an Azure Active Directory registered application in order to broker Log Analytics API access. See official documentation:
+https://docs.microsoft.com/en-us/power-apps/developer/data-platform/walkthrough-register-app-azure-active-directory
+
+Depending on your Azure subscription structure, it might be necessary to request this from a subscription administrator.
+
+#### b - Provide AAD registered application inforation
+HPCC logAccess requires access to the AAD Tenant, client, token, and target workspace ID via secure secret object.
+The secret is expected to be in the 'esp' category, and be named 'azure-logaccess'.
+The following kv pairs are supported
+- aad-tenant-id
+- aad-client-id
+- aad-client-secret
+- ala-workspace-id
+
+Example secret creation command (assuming ./secrets-templates contains a file named exactly as the above keys):
+```console
+  kubectl create secret generic azure-logaccess --from-file=HPCC-Platform/helm/examples/azure/log-analytics/secrets-templates/
+```
 
-This means functionality which fetches HPCC component logs will attempt to gather logs via KQL queries.
+#### c - Configure HPCC logAccess
+The target HPCC deployment should be directed to target the above Azure Log Analytics workspace by providing appropriate logAccess values (such as ./loganalytics-hpcc-logaccess.yaml). The previously created azure-logaccess secret must be declared and associated with the esp category, this can be accomplished via secrets value yaml (such as ./loganalytics-logccess-secrets.yaml)
 
 Example use:
 ```console
-  helm install myhpcc hpcc/hpcc -f HPCC-Platform/helm/examples/azure/log-analytics/loganalytics-hpcc-logaccess.yaml
+  helm install myhpcc hpcc/hpcc -f HPCC-Platform/helm/examples/azure/log-analytics/loganalytics-hpcc-logaccess.yaml -f  HPCC-Platform/helm/examples/azure/log-analytics/loganalytics-logccess-secrets.yaml
```

Comment:
@afishbeck can you confirm this is the intended approach for declaring jsecret secrets?
I'm concerned the secrets/vaults structure might be lead to inadvertent overwrites of a given secret/vault category (in this case 'esp')